src/CBT.NuGet/Tasks/NuGetRestore.cs

```diff
@@ -2,6 +2,7 @@
 using Microsoft.Build.Framework;
 using Microsoft.Build.Utilities;
 using System;
+using System.Diagnostics;
```

Comment:
del

Reply:
For whatever reason, `FileVersionInfo` is in `System.Diagnostics` namespace?